ccloud/docs/beginner-cloud.rst

```diff
@@ -233,29 +231,23 @@ Produce and consume records with Confluent Cloud CLI
 
       ccloud kafka topic create demo-topic-1
 
-#. Produce 10 messages to topic ``demo-topic-1`` by running the following
-   commands:
+#. Start producing to this topic ``demo-topic-1`` by running the following command:
 
    .. code-block:: bash
 
-         (for i in `seq 1 10`; do echo "${i}" ; done) | \
-           ccloud kafka topic produce demo-topic-1
+      ccloud kafka topic produce demo-topic-1
 
-#. Verify your output resembles:
+#. The CLI waits for you to type data at the prompt, so type a few characters each on a new line. For example, type th enumbers 1 through 5
```

Comment:
```suggestion
#. The CLI waits for you to type data at the prompt, so type a few characters each on a new line. For example, type the numbers 1 through 5:
```

Reply:
Just to note, the start.sh script produces numbers 1-10.

Reply:
@awalther28 good catching.  Dropping script down to 5 as well.